js/components/controlBar.js

```diff
@@ -553,6 +557,29 @@ var ControlBar = createReactClass({
         </div>
       ),
 
+      chromecast: function() {
+        if (this.props.controller.state.chromecastAvailable !== true)
+        {
+          return null;
+        }
+
+        let castIcon = "chromecast-disconnected";
+        if (this.props.controller.state.chromecastConnected === true)
+        {
+          castIcon = "chromecast-connected";
+        }
+        return (
+        <ControlButton
+          {...commonButtonProps}
+          key={CONSTANTS.CONTROL_BAR_KEYS.CHROMECAST}
+          focusId={CONSTANTS.CONTROL_BAR_KEYS.CHROMECAST}
+          ariaHidden={true}
```

Comment:
Is it deliberate to hide this button from screen readers?

Reply:
No, I was copy pasting things till it worked. I'll change it